src/hera/_cli/__init__.py

```diff
@@ -0,0 +1 @@
+"""Hera's command line interface."""
```

Comment:
I suggest we leave this public as there might be uses for it. Thoughts?

Reply:
Hmm, I think it depends on the API of the CLI, but it should be passing through to public functions we'll add elsewhere. We can always go public if want too, which is my preference unless the "might" becomes definite